ui/desktop/app/templates/scopes/scope/projects/targets.hbs

```diff
@@ -6,40 +6,36 @@
       {{#unless session.acknowledged}}
         <DialogDetail
           @heading={{concat
-            (t 'resources.target.host-set.detail.title')
+            (t 'resources.target.messages.connection-details.title')
             ': '
-            session.id
+            target.id
           }}
           @dismiss={{route-action 'acknowledge' session}}
           @dismissButtonText={{t 'actions.ok'}}
-          as |dialogDetail|
+          as |dialog|
         >
-          <dialogDetail.notification
+          <dialog.notification
```

Comment:
I am not sure (yet) if the notification should live out of the dialog body. Initially I went this way to not have to override paddings from the dialog body, but I think we need to start working with this component to determinate, for now here is goo enough.